elfloader-tool/src/arch-arm/linker.lds

```diff
@@ -28,7 +28,7 @@ INSERT BEFORE .text;
 
 SECTIONS
 {
-    .bss : {
+    .bss (NOLOAD) : {
```

Comment:
Can you add a note to this commit message that NOLOAD is already implicitly added by the current toolchain.